machine.go

```diff
@@ -397,11 +397,11 @@ func (m *Machine) setupLogging(ctx context.Context) error {
 	m.logger.Debug("Created metrics and logging fifos.")
 
 	l := models.Logger{
-		LogFifo:       m.cfg.LogFifo,
-		Level:         m.cfg.LogLevel,
-		MetricsFifo:   m.cfg.MetricsFifo,
-		ShowLevel:     true,
-		ShowLogOrigin: false,
+		LogFifo:       String(m.cfg.LogFifo),
+		Level:         String(m.cfg.LogLevel),
+		MetricsFifo:   String(m.cfg.MetricsFifo),
+		ShowLevel:     Bool(true),
+		ShowLogOrigin: Bool(false),
```

Comment:
The commit message doesn't shed any light on why we need these changes. Are they actually required as part of updating to support v0.15.0?

Reply:
Yea, the newly generated swagger models changed those to be pointers. I think it has something to do with setting default values and required fields, ie if a field isn't set, nil, set to default value.